CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact sandstone module, newest changes on the top.
 
+## [unreleased]
+
+### Added
+
+- `sandstone/Slider` prop `noWheelEvent` to disable wheel event handler
```

Comment:
What about `noChangeByWheel` or simply `noWheel`? `noWheelEvent` indicates _an implementation manner_, and it looks better to name a prop to mean _a certain hehavior by some user input_.

Reply:
I prefer `noWheel`